crates/codegen/ebnf/src/production.rs

```diff
@@ -16,7 +16,7 @@ impl<T: EBNFWriter> EBNFWritable<T> for Production {
                 VersionMap::Versioned(versions) => {
                     for (version, scanner) in versions {
                         writer.write_line_start();
-                        writer.write_comment(&format!("(* v{} *) ", version));
+                        writer.write_comment(&format!("(* v{version} *) "));
```

Comment:
Unfortunately auto-merge strategy doesn't work well if the PRs are stacked on top of each other, as if the first one is merged, the later ones will contain changes from both, making it much harder to review.

If not blocking/urgent, I will wait until this is rebase.

Reply:
done